docs/product-manuals/clients/cli-client/get-started.md

```diff
@@ -114,7 +115,7 @@ If the deployment is successful you'll get the following output:
 }
 ```
 
-Important here is the `bpmnProcessId`, which you'll need for creating a new instance.
+**IMPORTANT**: You will need the `bpmnProcessId` to create a new instance.
```

Comment:
An alternative approach for items like this is admonitions - https://docusaurus.io/docs/markdown-features/admonitions

Do you think this warrants something more visually impactful? Or is bold & all caps good enough? We should get some guidance into the style guide about when you use one over the other.

Reply:
I was going to ask you about this -- fantastic!!! Bookmarked this page and do believe these admonitions add visual ease when separating important notes. Should I add these as I go throughout all Cloud docs or hold off?

Reply:
Added this into the style guide and tagged you :)

Reply:
Hold off on changing all Cloud docs for now because I don't think the colors are correct for our branding. Modify the few in this section and see what the color choices look like for Important & Hint admonitions.

Reply:
Sounds great. I think the :::note option is pretty neutral for now.